src/csharp/Grpc.Core/Internal/NativeExtension.cs

```diff
@@ -96,11 +96,34 @@ private static UnmanagedLibrary Load()
             var netCorePublishedAppStylePath = Path.Combine(assemblyDirectory, runtimesDirectory, GetNativeLibraryFilename());
             var netCoreAppStylePath = Path.Combine(assemblyDirectory, "../..", runtimesDirectory, GetNativeLibraryFilename());
 
-            // Look for all native library in all possible locations in given order.
+            // Look for the native library in all possible locations in given order.
             string[] paths = new[] { classicPath, netCorePublishedAppStylePath, netCoreAppStylePath};
             return new UnmanagedLibrary(paths);
         }
 
+        /// <summary>
+        /// Loads native extension and return native methods delegates.
+        /// </summary>
+        private static NativeMethods LoadNativeMethods()
+        {
```

Comment:
Comment here or in related code that the "unity" related code paths won't be used if gRPC is being used from an official package?